style-about.css

```diff
@@ -0,0 +1,230 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;  
+}
+
+
+.header { 
+  font-family: 'Space Mono', sans-serif;
+  margin-bottom: 0;
+  font-weight: 700;
+  font-size: 48px;
+  letter-spacing: -2.14286px;
+  line-height: 48px;
```

Comment:
line-height is unitless, should divide its value to font-size. in this case 48 / 48 = 1 => `line-height: 1`.